docs/components/integration-framework/introduction.md

```diff
@@ -46,8 +46,8 @@ In a [Self-Managed](/self-managed/about-self-managed.md) environment, you provid
 The Camunda [Connector SDK](./connectors/custom-built-connectors/connector-sdk.md) allows you to run Connectors using [runtime environments](./connectors/custom-built-connectors/connector-sdk.md#runtime-environments).
 You can consume the pre-packaged runtime environment to execute any set of Connectors as described in the [installation guide](/self-managed/platform-deployment/overview.md).
```

Comment:
The installation guide is not explicit about this IMO. I would rather link to http://localhost:3000/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/#runtime-logic.

Reply:
The installation guide actually is the only place that is specific on how to consume the runtime in Self-Managed, if you ask me. Since there are different options (Docker, Manual), you have to consult those sections specifically on details then.

Reply:
I think the best place to link to is actually http://localhost:3000/docs/next/self-managed/connectors-deployment/install-and-start. This covers backend and frontend for Connectors in Self-Managed.

Reply:
[49833b0](https://github.com/camunda/camunda-platform-docs/pull/1376/commits/49833b01270769edc560abe2c9ff202f16372efb)